infra/scripts/language/agents/order_status_plugin.py

```diff
@@ -0,0 +1,11 @@
+from semantic_kernel.functions import kernel_function
+
+"""
+Sample plugin for returning order status in a customer support system - this plugin states order status
+and is used with a chat completion agent in a handoff orchestration system.
+"""
+class OrderStatusPlugin:
+    @kernel_function
+    def check_order_status(self, order_id: str) -> str:
+        """Check the status of an order."""
+        return f"Order {order_id} is shipped and will arrive in 2-3 days."
```

Comment:
nit - just so that there's some `flavor`, can we pick a time to arrive based on a hash of the input, where max_shipping_delay=14days? If possible - it would be great to add some `flavor` to the other agents too, so that they aren't too simple and hard to modify.

This can be a separate PR though